### PR TITLE
Multiple tiling adjustments

### LINKED ISSUE #26: Tiled logic bug when provided with a full grid
*Please refer to our [FAQ](https://ark-analysis.readthedocs.io/en/latest/_rtd/faq.html) and look at our [known issues](https://github.com/angelolab/ark-analysis/issues) before opening a bug report.*

**Describe the bug**
There is a logic error in `alpineer.load_utils.load_tiled_img_data` for the case when the number of FOVs is the same as spots in the grid (i.e. there are no black spaces in the stitched image).

The code runs, but the images are instead stitched in acquisition order and not the intended tiled shape. 

Screenshot 2023-04-06 at 12 03 02 PM

**Expected behavior**
We want the images to stitch in the correct order. 

Screenshot 2023-04-06 at 12 07 01 PM

**Solution**
Remove the following lines of code from the `load_tiled_img_data` function. in alpineer Even if there are no blank spots in the image, we should just always load in the images one at a time to ensure they are put in the correct location within the array. 
```
    # no missing fov images, load data normally and return array
    if len(fov_list) == len(expected_fovs):
        if single_dir:
            img_xr = load_imgs_from_dir(
                data_dir,
                match_substring=channel,
                xr_dim_name="channels",
                trim_suffix="_" + channel,
                xr_channel_names=[channel],
            )
        else:
            img_xr = load_imgs_from_tree(
                data_dir, img_sub_folder, fovs=fov_list, channels=[channel]
            )
        return img_xr
```
This is probably happened because the same function is used by both toffy and ark; however, in ark the folders have already been renamed and could be loaded in alphabetical order without issue.

### PR DESCRIPTION
**Pull request by camisowers**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Addresses the changes necessary to enable multiple tile stitching.
Also closes #26.

**How did you implement your changes**

Made the adjustments outlined in angelolab/toffy#351 and #26.
Added testing logic for multiple prefixes.

**Remaining issues**

New release and bump to toffy.
Update these function uses in ark repo to reflect to new returns and logic.

### CODE REVIEW

File: tests/load_utils_test.py [hunk 1]
```diff
@@ -355,67 +355,76 @@ def test_load_imgs_from_dir():
...
+    # MULTIPLE TILES
+    for tile_names in [("Tile_1_", "Tile_2_"), ("", "named_tile_")]:
+        prefix_1, prefix_2 = tile_names
+        fov_names = [f"{prefix_1}R6C9", f"{prefix_2}R3C5"]
+        tiles = load_utils.get_tiled_fov_names(fov_names, return_dims=True)
+
+        # check first tile for 6x9
+        prefix, expected_fovs, rows, cols = tiles[0]
+        assert prefix == prefix_1[:-1] and (rows, cols) == (6, 9)
+        assert expected_fovs == ns.natsorted(
```
Comment (srivarra) [inline tests/load_utils_test.py:417]: `natsort` is such an amazing module.

**Review by srivarra — APPROVED** [review]
@camisowers Looks good! Should we make a new release after this PR gets merged in?

**Review by alex-l-kong — APPROVED** [review]
Looks good

**Comment by camisowers** [comment]
> @camisowers Looks good! Should we make a new release after this PR gets merged in?

Yes, definitely.

**Review by ngreenwald — APPROVED** [review]

**Pull request merged**